lib/Service/AutoConfig/MxRecord.php

```diff
@@ -56,4 +56,17 @@ public function query(string $host) {
 		// TODO: sort by weight
 		return $mxRecords;
 	}
+
+	public function stripSubdomain(string $domain): string {
+		$labels = explode('.', $domain);
+
+		$top = count($labels) >= 2 ? array_pop($labels) : '';
+		$second = array_pop($labels);
+
+		return $second . '.' . $top;
+	}
+
+	public function getAllRecords(array $mxHosts): array {
```

Comment:
I would say the method is not getting all records but sanitize a list of records. Can we find a better name for it?

Reply:
sure, getSanitizedRecords?